context/theme-context.tsx

```
@@ -15,41 +21,70 @@ type ThemeContextType = {
+function getServerSnapshot(): Theme {
   return 'light';
 }
 
+function subscribeToTheme(callback: () => void): () => void {
+  listeners.add(callback);
+  return () => listeners.delete(callback);
+}
+
+function setThemeValue(newTheme: Theme): void {
+  currentTheme = newTheme;
+  listeners.forEach((listener) => listener());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)